test/main.js

```diff
@@ -61,6 +61,25 @@ describe('glob-stream', function() {
       });
     });
 
+    it('should handle ( ) in directory paths', function(done) {
+      var cwd = join(__dirname, './fixtures/has (parens)');
```

Comment:
what happens if the `cwd` is `./fixtures` and you use `**/*.dmc` as the glob?  Maybe that should be another test